src/NATS.Client.Core/Commands/CommandWriter.cs

```diff
@@ -24,7 +24,7 @@ internal sealed class CommandWriter : IAsyncDisposable
     private const int SendMemSize = 8520;
 
     // set to a reasonable socket write mem size
```

Comment:
We should link this conversation in a comment so we know where the number came from

```suggestion
    // should be more than SendMemSize
    // https://github.com/nats-io/nats.net.v2/pull/383#discussion_r1484344102
```